airbyte-integrations/connectors/destination-google-sheets/destination_google_sheets/writer.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+from airbyte_cdk.models import AirbyteStream
+from pygsheets import Worksheet
+
+from .buffer import WriteBuffer
+from .spreadsheet import GoogleSheets
+
+
+class GoogleSheetsWriter(WriteBuffer):
```

Comment:
Would it be fair to say that WriteBuffer is functioning as a Mixin here? (rather than GoogleSheetsWriter being a type of WriteBuffer?) If you agree, I'd suggest renaming for clarity (e.g. `WriteBufferMixin`)

Reply:
Basically, I agree, it's the inheritance, just to include the methods inside the GoogleSheetsWriter, instead of calling them directly. Changed to be `WriteBufferMixin`